bin/disk/create/index.js

```diff
@@ -15,19 +15,23 @@ const options = {
         required: true,
     },
     type: {
-        description: 'Disk type ID or name',
+        description: 'Disk type ID or name. Required if no source disk is specified',
         type: 'string',
-        required: true,
     },
     size: {
-        description: 'Disk size in GiB. Required if no source file is specified',
+        description: 'Disk size in GiB. Required if no source file and no disk is specified',
```

Comment:
@fredericosilva , which one?
* Disk size in GiB. Required if no source file or no source disk is specified.
* Disk size in GiB. Required if no source file nor source disk is specified.